libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -353,6 +353,11 @@
 #define HAL_FLOW_PX4_MAX_FLOW_PIXEL 4
 #define HAL_FLOW_PX4_BOTTOM_FLOW_FEATURE_THRESHOLD 30
 #define HAL_FLOW_PX4_BOTTOM_FLOW_VALUE_THRESHOLD 5000
+#define HAL_GPIO_A_LED_PIN        69
```

Comment:
Use this https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_Linux/GPIO_BBB.h#L44 instead of 69.